src/aiokeycloak/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .client import KeycloakClient
+from .factory import KeycloakClientFactory
+from .errors import KeycloakError, UnauthorizedError, UserExistsError
```

Comment:
It's unnecessary

Reply:
add `__all__`

Reply:
I has modify by your request on branch develop